React/Views/RCTRefreshControl.m

```diff
@@ -119,9 +113,7 @@ - (void)setRefreshing:(BOOL)refreshing
     if (refreshing) {
       // If it is the initial render, beginRefreshing will get called
       // in layoutSubviews.
```

Comment:
These comments don't make sense when you short-circuit the IF condition.  They should be removed/modified.